pages/windows/assoc.md

```diff
@@ -1,16 +1,17 @@
 # assoc
 
 > Display or modify file extension associations.
+> See also: `ftype`.
 > More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/assoc>.
 
-- Display all associated filetypes:
+- Print all associated filetypes:
 
 `assoc`
 
-- Display the associated filetype for a specific extension:
+- Print the associated filetype for the specified extension (.txt):
 
 `assoc {{.txt}}`
 
-- Modify the associated filetype for a specific extension:
+- Set/remove the associated filetype (txtfile) for the specified extension (.txt):
 
-`assoc {{.txt}}={{txtfile}}`
+`assoc {{.txt}}={{txtfile| }}`
```

Comment:
What's this?
```suggestion
`assoc {{.txt}}={{txtfile}}`
```

Reply:
It's a valid syntax according to [docs][docs]:

> To remove the file type association for a file name extension, add a white space after the equal sign by pressing the SPACEBAR.

[docs]: https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/assoc#remarks